src/instructions/tables.rs

```diff
@@ -34,7 +34,7 @@ pub unsafe fn lgdt(gdt: &DescriptorTablePointer) {
 /// ## Safety
 ///
 /// This function is unsafe because the caller must ensure that the given
-/// `DescriptorTablePointer` points to a valid IDT and that loading this
+/// `DescriptorTablePointer` points to a valid LIDT and that loading this
```

Comment:
What are these changes about? The table that is loaded is an `IDT`, so I think that this was correct previously...